src/System.Management.Automation/engine/parser/ast.cs

```diff
@@ -7891,14 +7909,26 @@ public MemberExpressionAst(IScriptExtent extent, ExpressionAst expression, Comma
         /// </summary>
         public bool NullConditional { get; protected set; }
 
+        /// <summary>
+        /// List of generic type arguments passed to this member.
+        /// </summary>
+        public ReadOnlyCollection<ITypeName> GenericTypeArguments { get; private set; }
```

Comment:
Is the `private` setter used anywhere?

Reply:
Doesn't appear to be used anymore, I'll take it out. Think it was being set after construction in an earlier version of this code.